src/utils/date.ts

```diff
@@ -1,25 +1,27 @@
 import { DateTime } from 'luxon';
 
-export const parseAsDate = (date: string) =>
-  DateTime.fromISO(date, { zone: 'utc' });
+// Used to parse out mintute, day, ,month, and years from ISOString
+export const parseAsUTC = (date: string) => DateTime.fromISO(date).toUTC();
 
-export const parseAsLocalTime = (date: string) => DateTime.fromISO(date);
-
-export const formatDateAndIgnoreTimezone = (date: string) =>
-  parseAsDate(date).toFormat('MMMM d yyyy');
-
-export const formatDate = (date: string | DateTime) => {
-  // ISO String
-  if (typeof date === 'string') {
-    return parseAsLocalTime(date).toFormat('MMMM d yyyy');
-  }
-
-  // luxon DateTime
-  if (date instanceof DateTime) {
-    return date.toFormat('MMMM d yyyy');
-  }
+type DateParserType = {
+  date: string;
+  serverGenerated: boolean;
+  format?: 'MM/dd/yyyy' | 'MMMM d, yyyy';
+};
 
-  return '';
+/*
+ Currently only alternative in use to MMMM d yyyy is MM/dd/yyyy
+ User input date need to be parsed in UTC to preserve date
+ If iso string contains specific time (server generated dates - ex. dateSubmitted), parse locally
+*/
+export const formatDate = ({
```

Comment:
The formatDate util can be simplied into 2 functions instead of coupling behavior on the "serverGenerated" oddity. Use function comments to explain which one to use for "server generated" vs "user input." Something like

```
type DateFormat = ...

/** 
 * Output local timezoned dates from iso string. 
 * Typically used for dates that handle iso strings correctly. 
 */
formatDate(date: string, format: DateFormat) {
  return DateTime.fromISO(date).toFormat(format);
}

/** 
 * Output UTC timezoned dates from iso string. 
 * Typically used for dates from user input, where utc timezone needs to be set 
 * explicitly in order to match timezoneless dates within a iso string correctly.
 */
formatDateUtc(date: string, format: DateFormat) {
  return DateTime.fromISO(date, {setZone: true}).toFormat(format);
}
```